test/functional/Valhalla/playlist.xml

```diff
@@ -26,8 +26,11 @@
 		<testCaseName>ValueTypeTests</testCaseName>
 		<variations>
 			<variation>NoOptions</variation>
-			<variation>-Xgcpolicy:nogc -XX:ValueTypeFlatteningThreshold=99999 -XX:+EnableArrayFlattening</variation>
-			<variation>-Xgcpolicy:optthruput -Xnocompactgc -XX:ValueTypeFlatteningThreshold=99999 -XX:+EnableArrayFlattening</variation>
+			<!--
```

Comment:
This isn't what I meant. You can disable the specific tests that fail. Look for `@Test(enabled = false, priority=5)` in the tests